cl/recap/tasks.py

```diff
@@ -1378,6 +1378,7 @@ def fetch_docket(self, fq_pk):
     :param fq_pk: The PK of the RECAP Fetch Queue to update.
     :return: None
     """
+    start_time = now()
```

Comment:
Isn't it better to put this as close to the actual database manipulation as possible? Or if not near the DB manipulation, just before we request the data from PACER?

Reply:
Changed, I put this just before we request the docket from PACER.